integration_test/gce/gce_testing.go

```diff
@@ -739,6 +739,95 @@ func runCommand(ctx context.Context, logger *log.Logger, stdin io.Reader, args [
 	return output, err
 }
 
+// getGcloudConfigDir returns the current gcloud configuration directory.
+func getGcloudConfigDir(ctx context.Context) (string, error) {
+	null, err := os.Open(os.DevNull)
+	if err != nil {
+		panic(fmt.Sprintf("unable to open os.DevNull: %v", err))
```

Comment:
why not just return the error normally instead of panicing? panicing causes stuff like test cleanup to not run, which I guess isn't so bad now that we have automatic VM cleanup, but anyway. Maybe it causes stuff to show up worse in ResultStore? I'd rather just have a normal error be returned.

Reply:
This was one of those "should never ever happen" scenarios. However, it's even better to use `io.Discard`, so done.